requirements.txt

```diff
@@ -18,4 +18,7 @@ pandas == 2.0.0
 nltk == 3.8.1
 spacy == 3.7.4
 en_core_web_sm @ https://github.com/explosion/spacy-models/releases/download/en_core_web_sm-3.7.1/en_core_web_sm-3.7.1-py3-none-any.whl
-rouge_score == 0.1.2 
+rouge_score == 0.1.2
```

Comment:
add the required packages in the setup.py as in the requirements.txt

Reply:
> add the required packages in the setup.py as in the requirements.txt

Added `benchmarks` field in `extras_requires`.